src/cockpit/389-console/src/lib/tools.jsx

```diff
@@ -75,7 +75,7 @@ export function get_date_string(timestamp) {
         parseInt(minute),
         parseInt(sec)
     );
-    return date.toLocaleString();
+    return date.toLocaleString('en-ZA');
```

Comment:
Hm, is there any particular reason for 'en-ZA' locale? (ZA part confuses me)

Reply:
Good question.  So in the monitor the server start date uses MM/DD/YYYY, but I want it to use a more universal format.  Locale "en-ZA" uses YYYY/MM/DD   I'm open to other suggestions as well.

Reply:
Agree, it's a nice format. You have my full ack :)